source/crud/read-operations/text.txt

```diff
@@ -0,0 +1,260 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+Use the `$text
+<https://docs.mongodb.com/manual/reference/operator/query/text/>`_
+operator to perform text searches on fields which have a
+`text index <https://docs.mongodb.com/manual/core/index-text/>`_.
+
+Overview
+--------
+
+Text searches let you search a collection of strings for words or
+phrases, instead of matching substrings like
+`regular expresssions
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions>`_,
+also known as RegEx. While regex is very powerful, and MongoDB even
+offers a `$regex <>` operator, text searches work particularly well when
+applied to unstructured text, like transcripts, essays, or even web
+pages. Because MongoDB's text indexes are aware of concepts like
+plurality, case sensitivity, stop words, whitespace, and punctuation in
+multiple languages, all you have to do is query with a search term to
+find documents that contain your term.
+
+You can only create *one* text index per collection, so every text
+search queries all fields used by the index for term matches.
```

Comment:
This is important. Can we put it in an important admonition?

Reply:
Agreed.